py/sea/commands.py

```diff
@@ -1614,6 +1614,88 @@ def run (self, args, extra):
+        super(CexGen, self).__init__("cex",
+                            'Generate Counter-example executable from harness',
+                            allow_extra=True)
+        self.clangCmd = None
+
+    @property
+    def stdout(self):
+        return self.clangCmd.stdout
+
+    def name_out_file(self, in_files, args=None, work_dir=None):
+        return _remap_file(in_files[0], '', work_dir)
+
+    def mk_arg_parser(self, ap):
+        ap = super(CexGen, self).mk_arg_parser(ap)
+        add_tmp_dir_args(ap)
+        add_in_out_args(ap)
+        _add_S_arg(ap)
+
+        ap.add_argument('--run', '-run', dest='run',
+                        help='Run generated executable',
+                        default=False, action='store_true')
+        ap.add_argument('--klee-internalize', '-klee-internalize',
+                        dest='internalize',
+                        help='Apply klee internalize pre-processing',
+                        default=False, action='store_true')
+        ap.add_argument('--only-strip-extern', '-only-strip-extern',
+                        dest='strip_extern',
+                        help='Strip external functions during internalize pre-processing',
+                        default=False, action='store_true')
+        ap.add_argument('--keep-lib-fn', '-keep-lib-fn',
+                        dest='keep_lib_fn',
+                        help='Keep lib functions during internalize pre-processing',
+                        default=False, action='store_true')
+        ap.add_argument('--wrap-mem', '-wrap-mem',
+                        dest='wrapmem',
+                        help='Wrap memory read/access instructions',
+                        default=False, action='store_true')
+        ap.add_argument('--rmtf', '-rmtf',
+                        dest='rmtf',
+                        help='Remove target features for cross platform files',
+                        default=False, action='store_true')
+        return ap
+
+    def run(self, args, extra):
+        cmds = [Clang()]
+        # following logic in Seapp, where klee internalize is separate
+        if args.strip_extern:
+            pp = Seapp(strip_extern=True, keep_lib_fn=args.keep_lib_fn)
+            cmds.append(pp)
+        if args.internalize:
+            cmds.append(Seapp(internalize=True))
+
+        if args.wrapmem:
+            cmds.append(WrapMem())
+
+        if args.rmtf:
+            cmds.append(RemoveTargetFeatures())
+        
+        if args.out_file is None:
+            target = os.path.basename(args.in_files[0]).split(sep='.')[0]
+            args.out_file = "/tmp/{}_debug.exe".format(target)
+
+        linkRtCmd = LinkRt()
+        self.clangCmd = linkRtCmd
+        cmds.append(linkRtCmd)
+
+        pipeline = sea.SeqCmd('', '', cmds)
+        argv = ['-m64', '-g']
+        argv.extend(extra)
+        res = pipeline.run(args, argv)
+        if res:
+            print("\n Executable Counterexample generation failed...")
```

Comment:
there should not be a new line before print. Only after. If it is required, usually indicates an issue elsewhere